src/sqlfluff/core/linter/linter.py

```diff
@@ -519,6 +519,14 @@ def render_string(
         # Start the templating timer
         t0 = time.monotonic()
 
+        if not config.get("templater_obj") == self.templater:
+            linter_logger.warn(

```

Comment:
@NiallRees this is the warning we discussed

Reply:
This is nice 👍